Makefile

```diff
@@ -26,9 +26,9 @@ ifneq ($(GOSUM),)
   ldflags += -X github.com/cosmos/cosmos-sdk/version.VendorDirHash=$(shell $(GOSUM) go.sum)
 endif
 
-ifeq ($(WITH_CLEVELDB),yes)
+ifeq ($(WITH_ROCKSDB),yes)
   build_tags += gcc
-  ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=cleveldb
+  ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=rocksdb
```

Comment:
I think that by doing this we are not using RocksDB as the default DB backend (as specified inside #111). Instead, we are allowing people to use it if they want when compiling the Desmos binaries. 

I think the correct way of implementing #111 is by reverting these changes and changing the original

```
ifeq ($(WITH_CLEVELDB),yes)
  build_tags += gcc	 
  ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=cleveldb
endif
```

To be 

```
ifeq ($(DB_BACKEND),cleveldb)
  build_tags += gcc	 
  ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=cleveldb
else ifeq($(DB_BACKEND),leveldb)
  // Do nothing as LevelDB is the default one inside Cosmos SDK
else
  build_tags += gcc	 
  ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=rocksdb
endif
```

This would probably make RocksDB the default system in our case while still allowing users to go back to LevelDB or CLevelDB